src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -123,14 +130,47 @@ def __init__(self, **kwargs):
         self.xy_move_widget = widget_xy_move_drywall.XYMoveDrywall(machine=self.m, screen_manager=self.sm)
         self.xy_move_container.add_widget(self.xy_move_widget)
 
+        self.drywall_shape_display_widget = widget_drywall_shape_display.DrywallShapeDisplay(machine=self.m, screen_manager=self.sm, dwt_config=self.dwt_config)
+        self.shape_display_container.add_widget(self.drywall_shape_display_widget)
+
+        self.shape_selection.text = 'circle'
+
     def home(self):
         self.m.request_homing_procedure('drywall_cutter', 'drywall_cutter')
 
     def select_shape(self):
-        pass
-
-    def rotate_shape(self):
-        pass
+        if self.shape_selection.text in ['line', 'geberit']:
+            # Only on line available for these options
+            self.cut_offset_selection.text = 'on'
+            self.cut_offset_selection.disabled = True
+        else:
+            # Default to cut inside line
+            self.cut_offset_selection.text = 'inside'
+            self.cut_offset_selection.disabled = False
+
+        if self.shape_selection.text in ['rectangle', 'line']:
+            self.rotate_button.disabled = False
+        else:
+            self.rotate_button.disabled = True
+
+        self.rotation = 'horizontal'
```

Comment:
This hardcoded default may also cause issues - see next comment :)

Reply:
I have updated this section of the code in newer branches to account for this: https://github.com/YetiTool/easycut-smartbench/blob/drywall_break_testing/src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py#L176-L177